web/ui/react-app/package.json

```diff
@@ -38,18 +38,15 @@
     "react-copy-to-clipboard": "^5.0.1",
     "react-dom": "^16.7.0",
     "react-resize-detector": "^5.0.7",
-    "react-scripts": "3.4.4",
-    "react-test-renderer": "^16.9.0",
     "reactstrap": "^8.9.0",
     "sanitize-html": "^1.20.1",
     "tempusdominus-bootstrap-4": "^5.1.2",
     "tempusdominus-core": "^5.0.3",
-    "typescript": "^3.3.3",
     "use-media": "^1.4.0"
   },
   "scripts": {
     "start": "react-scripts start",
-    "build": "react-scripts build",
+    "build": "CI= react-scripts build",
```

Comment:
I don't think we should add the `CI=` here... the only time where we want it is in Netlify, and doing that via the build command line in there (that contains the `sed` as well).

Reply:
so we need to fix all warnings raised then. It's fine it will just take a bit more time

Reply:
Hmm, yeah, sorry, I guess I'm too tired today. I actually remember having this issue in PromLens as well, and I fixed a bunch of similar style warnings there to make things work when updating react-scripts to 4.0. Probably either fixing the warnings or adding explicit exceptions in some cases would be the best thing to do.

Reply:
I finally fixed all warning, but now it seems fetch doesn't exist anymore which is weird. It should still be injected by `jest-fetch-mock`.